<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Navigator</class>
 <widget class="QWidget" name="Navigator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(51,153,255);	
	border-color: rgb(0, 0, 0);
}

QPushButton
{
	border-radius:9px;
	background-color:white;
	font:20px, &quot;SimHei&quot;;
	outline:none;
}</string>
  </property>
  <widget class="QLabel" name="labelBackground">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRunstatePage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>5</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNetworkPage">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>5</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>网络</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBypass">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>5</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>旁路</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConfig">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>5</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRecondition">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>5</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>检修</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEvent">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>5</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>事件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHelp">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>2</y>
     <width>65</width>
     <height>65</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/help.png);
background-color:rgb(51,153,255);
font:20px, &quot;SimHei&quot;;
outline:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
